files/en-us/web/css/forced-color-adjust/index.html

```diff
@@ -0,0 +1,125 @@
+
+<h3 id="Values">Values</h3>
+
+<dl>
+	<dt><code>auto</code></dt>
+	<dd>The element's colors are adjusted by the {{Glossary("user agent")}} in forced colors mode. This is the default.</dd>
+	<dt><code><strong>none</strong></code></dt>
+	<dd>The element's colors are not automatically adjusted by the {{Glossary("user agent")}} in forced colors mode.</dd>
+</dl>
+
+<h2 id="Usage_notes">Usage notes</h2>
+
+<p>This property should only be used to makes changes that will support a user's color and contrast requirements. For example, if you become aware that the color optimizations made by the {{Glossary("user agent")}} result in a poor experience when in a high contrast or dark mode. Using this property would enable tweaking of the result in that mode to provide a better experience. It should not be used to prevent user choices being respected.</p>
+
+<p>The property is a standardized version of the <a href="https://docs.microsoft.com/en-us/previous-versions/hh771863%28v%3dvs.85%29">-ms-high-contrast-adjust</a> property. The prefixed property can be used to achieve similar functionality in Internet Explorer 10 and EdgeHTML versions of Microsoft Edge.</p>
+
+<h2 id="Formal_definition">Formal definition</h2>
+
+<p>{{cssinfo}}</p>
+
+<h2 id="Formal_syntax">Formal syntax</h2>
+
+<pre class="syntaxbox notranslate">{{csssyntax}}</pre>
+
+<h2 id="Examples">Examples</h2>
+
+<h3 id="Preserving_colors">Preserving colors</h3>
+
+<p>In the example below the first box will use the color scheme that the user has set. For example in Windows High Contrast mode black scheme it will have a black background and white text. The second box will preserve the site colors set on the <code>.box</code> class.</p>
+
+<p>By using the <code><a href="/en-US/docs/Web/CSS/@media/forced-colors">forced-colors</a></code> media feature you could add any other optimizations for forced color mode alongside the <code>forced-color-adjust</code> property.</p>
+
+<h4 id="CSS">CSS</h4>
+
+<pre class="brush: css notranslate">.box {
+  border: 5px solid grey;
+  background-color: #ccc;
+  width: 300px;
+  margin: 20px;
+  padding: 10px;
+}
+
+@media (forced-colors: active) {
+  .forced {
+    forced-color-adjust: none;
+  }
+}</pre>
+
+<h4 id="HTML">HTML</h4>
+
+<pre class="brush: html notranslate">&lt;div class="box"&gt;
+  &lt;p&gt;This is a box which should use your color preferences.&lt;/p&gt;
+&lt;/div&gt;
+
+&lt;div class="box forced"&gt;
+  &lt;p&gt;This is a box which should stay the colors set by the site.&lt;/p&gt;
+&lt;/div&gt;</pre>
+
+<h4 id="Result">Result</h4>
+
+<p>{{EmbedLiveSample("Preserving_colors", 640, 260)}}</p>
+
+<figure><img src="windows-high-contrast.jpg" alt="The example above in high contrast mode shows the first box with a black background the second with the grey background of the CSS.">
+<figcaption>The example above in Windows High Contrast Mode</figcaption></figure>
```

Comment:
```suggestion
<figure role="figure" aria-label="The example above in Windows High Contrast Mode"><img src="windows-high-contrast.jpg" alt="The example above in high contrast mode shows the first box with a black background the second with the grey background of the CSS.">
<figcaption>The example above in Windows High Contrast Mode</figcaption></figure>
```

This will help with some screen reader announcement issues.

Reply:
cool - would be good to add stuff like this to the docs for adding images (unless it already is and I completely ignored it!)